src/routes/forgot-password/forgot-password.js

```diff
@@ -1,21 +1,40 @@
 import React, { Component } from 'react';
 import { Grid } from 'semantic-ui-react';
 import { ForgotPasswordForm } from 'components/forms';
-import { SubmissionError } from 'redux-form';
 import { userService } from 'api';
 import PropTypes from 'prop-types';
 
+function getErrorMessage(response) {
+  if (!response) {
+    return 'Unknown error occured';
+  }
+
+  const { status } = response;
+  switch (status) {
+    case 404:
+      return 'Page not found';
```

Comment:
if page is not found, shouldn't the user be shown a 404 page? if there is a 404 in the response the url is wrong.